api/apps/__init__.py

```
@@ -178,10 +176,10 @@ def _load_user(auth_types=None):
                     g.auth_type = AUTH_BETA
                     g.user = user[0]
                     return user[0]
-            g.auth_error_message = 'Authentication error: API key is invalid! '
+            g.auth_error_message = "Authentication error: API key is invalid! "
         except Exception as e_beta:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)